todoman/cli.py

```diff
@@ -48,6 +48,23 @@ def _validate_date_param(ctx, param, val):
         raise click.BadParameter(e)
 
 
+def _validate_start_date_param(ctx, param, val):
+    try:
+        if 'before' in val:
```

Comment:
I'd prefer `val.startswith('before ')`, because you're going to have a very weird error message when somebody enters something like `2017-09-09 before`.